README.md

```diff
@@ -0,0 +1,56 @@
+# dnssnoop
+
+dnssnoop utility for tracing standard DNS queries as they are sent from your system to DNS servers
+from all processes on a Linux system in real-time. It captures information related to both the
+query made and the resulting response with minimal system overhead.
+
+## Deploying
+
+dnssnoop is a yeet that can be deployed to a yeet daemon using the yeet package manager or directly
+from the system running the yeet daemon.
+
+### From YPM
+
+Go to <https://yeet.cx/@yeet/dnssnoop>, select the host on which you'd like it deployed, and click
+"Deploy".
+
+### From Source
+
+Build the eBPF program:
+
+    make
+
+Deploy to the local host:
+
+    yeet add .
+
+## Data Schema
+
+Each time a DNS query is made and answered an event is emitted:
+
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| pid | `KernelPid` | The **thread** ID of the thread that made this query. |
+| tgid | `KernelPid` | The **process** ID of the process that made this query. |
+| cgroup_id | `INT` | The ID of the control group associated with the process that made this query. |
+| latency_ns | `INT` | The latency, in nanoseconds, between the request and the reply. |
+| id | `INT` | The transaction ID of the query. |
+| comm | `STRING` | The name of the thread that made this query. |
+| name | `STRING` | The domain name being queried. |
+| c_group | `STRING` | The name of the control group associated with the process that made this query. |
+| server_ip | `INT` | The IP address of the DNS server. |
+
+### Example Query
+
+You can query a collection from this yeet in the query editor with a query like:
```

Comment:
nice. should we add some useful ones suggested by like a gpt to give ppl ideas of how they can use it?

and fix the readme to reflect the changes you made.

Reply:
Updated.

Did not add extra examples, I think you have a better idea of what those should look like.